wp-content/themes/core/components/card/css/card.pcss

```diff
@@ -179,4 +243,27 @@
 	.c-card--style-outlined & {
 		padding: 0 var(--spacer-30);
 	}
+
+	/* CASE: Style: Search */
+	.c-card--style-search & {
+		margin-top: 0;
+	}
+}
+
+/* -----------------------------------------------------------------------------
+ * Card Call To Action Link
+ * ----------------------------------------------------------------------------- */
+
+.c-card__cta-link {
+
+	/* CASE: Style: Search */
+	.c-card--style-search & {
+		word-break: break-all; /* Break long URLs */
+
+		&:focus,
+		&:hover,
+		.c-card--style-search:hover& {
```

Comment:
I _think_ there's a space missing before the ampersand?

Reply:
Yea, this selector is kind of weird, but I had to do that to access the hover state of the parent `.c-card---style-search`. It builds a funky selector, but it is nice to keep the attribute change on one line. Any thoughts of how to clean that up @dpellenwood?

Reply:
Ah, gotcha. You're g2g with this as-is. Maybe just a comment for future-us?